tidb-cloud/tidb-cloud-security-faq.md

```diff
@@ -0,0 +1,91 @@
+
+Customer data includes data that customers or their consumers store in TiKV or TiFlash, which is located in the cloud providers' storage (such as S3 buckets or EBS of AWS).
+
+Customer data does not include TiDB Cloud operational data, diagnosis data, or metadata:
+
+- Operational data: TiDB Cloud control plane logs, runtime metrics, usage policies, and service ticket-related information.
+- Diagnosis data: TiDB cluster logs (including logs, error logs, and slow query logs), metrics (including usage and runtime metrics), configuration (including TiDB cluster configuration items, node IP addresses, and values), monitoring, and alert-related information.
+- Metadata: TiDB Cloud account registration information (including email, company name, phone number, and IP address), permissions, endpoint services, devices, versions, and tag-related information.
+
+For more information about the operational data, see [TiDB Cloud Privacy Policy](https://www.pingcap.com/privacy-policy/) and [TiDB Cloud Data Processing Agreement](https://www.pingcap.com/legal/data-processing-agreement-for-tidb-cloud-services/).
+
+## Who owns customer data?
+
+As a customer, you have full ownership of your data and can choose which TiDB Cloud cluster can process, store, and host your data. TiDB Cloud will not access or use your data without your permission.
+
+## Who has control over customer data?
+
+As a customer, you have complete control over your data:
+
+- You can choose how to protect your data using TiDB Cloud, which offers encryption at rest and in transit to safeguard your data. Additionally, you have the option to manage your encryption key. If you prefer to bring your key, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).
+
+- You can manage access to your data, as well as the TiDB Cloud databases and tools, through users, roles, and credentials that you control.
+
+## Is customer data securely stored in TiDB Cloud?
+
+In TiDB Cloud, customer data is protected through a combination of features and best practices, including:
+
+- TiDB Cloud provides encryption for data both at rest and in transit. This means that the data is encrypted when it is stored on disks and when it is transmitted over the network.
+
+- TiDB Cloud allows customers to set up an IP access list to control who has access to their data, and also allows customers to set up roles and assign privileges to specified users only.
+
+- TiDB Cloud runs in a virtual private cloud (VPC) that isolates customer traffic from other traffic on the internet, and TiDB Cloud supports secure network communication using SSL/TLS.
+
+- TiDB Cloud provides various backup and disaster recovery options to ensure that customer data is protected in case of an outage or disaster. Customers can set up automatic backups, incremental backups, and point-in-time recovery (PITR), and replicate their data to different regions for disaster recovery.
```

Comment:
I would suggest removing all data safety (not security) related stuff from this section. Mixing disaster recovery mechanisms with security practices is not good.